docs/data-model.rst

```diff
@@ -993,15 +993,18 @@ is ``7``.
 Missing data
 ************
 
-Missing data is encoding in tskit using the idea of *isolated samples*. An
-isolated sample is a sample node (see :ref:`sec_data_model_definitions`)
-that has no children and no parent, in a particular tree. This is encodes
-the idea that we don't know anything about a given sample over a specific
-interval. This definition covers the standard idea of missing data in
-genomics (where we do not know the sequence of a given contemporary sample at
-some site, for whatever reason), but also more generally the idea that
-we may not know anything about large sections of the genomes of ancestral
-samples.
+Missing data is encoding in tskit using the idea of *isolated samples*.
+A sample's genotype is missing at a position if it is *isolated* and if it has
```

Comment:
Should we use "site" rather than position?

Reply:
Gah, a good question. Could be either, but I think that "position" is the position along the genome, and so that's where the genotype is (or is not known), while "site" is a special object that might also be associated with the position. For instance, ts.site( ) doesn't get you genotypes, but ts.variants( ) does (although the site is one of the things returned). That's why I wrote 'position'?

Reply:
Cool, I'm happy with position as you can only have one site per position anyway.